internal/env/env.go

```diff
@@ -0,0 +1,37 @@
+package env
```

Comment:
Ideally package names should not use words that we would frequently use as variables or functions (because then it forces us to use more contrived names for those). In addition, using made-up or composite words for packages makes them easy to identify.

The reason we use `envir` was precisely this.

Reply:
Good point about avoiding words frequently used as variables.

Personally, I'd prefer `environ` over `envir`, since `envir` is not a common substitute for `environment`, and harder to understand what it may stand for if seen in isolation.

Reply:
Okay, I'll create a PR to rename the package. Which name is better? `envir` or `environ`?

Reply:
eh, lets go with `envir` so a bit less to type?

Reply:
See https://github.com/jetpack-io/devbox/pull/962